rust_src/src/lisp.rs

```diff
@@ -899,6 +899,17 @@ impl From<LispWindowRef> for LispObject {
     }
 }
 
+impl From<LispObject> for Option<LispWindowRef> {
+    #[inline]
+    fn from(o: LispObject) -> Self {
```

Comment:
We pay for this check twice. Why not use as_window?

Reply:
True. Maybe we should start introducing result types now ?